### ISSUE TITLE
Character sheet doesn't function properly

### ISSUE THREAD

**Issue by operinko** [kakaroto/fvtt-module-popout#1]
The sheet pops out properly, but nothing on it is clickable.
A few errors in the console:
```
foundry.js:2023 Foundry VTT | Rendering ActorSheet5eCharacter
...
foundry.js:2967 Foundry VTT | Pushed compiled template templates/dice/roll.html to cache.
foundry.js:22702 Uncaught (in promise) TypeError: Cannot read property 'scrollHeight' of undefined
    at ChatLog.scrollBottom (foundry.js:22702)
    at foundry.js:22717
scrollBottom @ foundry.js:22702
(anonymous) @ foundry.js:22717
async function (async)
postAll @ foundry.js:22754
_render @ foundry.js:22471
async function (async)
_render @ foundry.js:22470
render @ foundry.js:2005
_render @ foundry.js:12227
async function (async)
_render @ foundry.js:12224
render @ foundry.js:2005
initializeUI @ foundry.js:3295
setupGame @ foundry.js:3243
_initializeGame @ foundry.js:3697
initialize @ foundry.js:3222
async function (async)
initialize @ foundry.js:3215
(anonymous) @ foundry.js:3795
foundry.js:2967 Foundry VTT | Pushed compiled template public/systems/dnd5e/templates/items/equipment-details.html to cache.
```
```
foundry.js:2967 Foundry VTT | Pushed compiled template public/systems/dnd5e/templates/items/spell-details.html to cache.
VM2439:1 Uncaught SyntaxError: Unexpected identifier
setTimeout (async)
(anonymous) @ VM2382 game:1
_call @ foundry.js:1111
callAll @ foundry.js:1076
setupGame @ foundry.js:3257
async function (async)
setupGame @ foundry.js:3244
_initializeGame @ foundry.js:3697
initialize @ foundry.js:3222
async function (async)
initialize @ foundry.js:3215
(anonymous) @ foundry.js:3795
foundry.js:2967 Foundry VTT | Pushed compiled template public/systems/dnd5e/templates/items/spell-sidebar.html to cache.
```
```
foundry.js:2967 Foundry VTT | Pushed compiled template public/systems/dnd5e/templates/items/weapon-sidebar.html to cache.
foundry.js:1009 Something went wrong with FontAwesome
```

**Comment by Oromis** [comment]
Seems to work for me now (on 0.4.3). Has this been fixed?

**Comment by kakaroto** [comment]
Just noticed these issues, sorry! For some reason, this repository wasn't watched so I never got notifications when the issues were created.
Yes, that was fixed in 0.4.3. Thanks for being patient.
